cpp/sequel.cpp

```diff
@@ -306,7 +306,7 @@ SequelResult sequel_execute(jsi::Runtime &rt, string const dbName, string const
 
         case SQLITE_INTEGER:
         {
-          int column_value = sqlite3_column_int(statement, i);
+          double column_value = sqlite3_column_double(statement, i);
           entry.setProperty(rt, column_name.c_str(), jsi::Value(column_value));
```

Comment:
@ospfranco The value is passed as a `jsi::Value`, and there's no constructor for `jsi::Value` that can accept an `int64_t` on purpose. JS can't represent all values, but it can represent all ints up to 53 bits wide (as a double). It won't cause garbage values.

Reply:
yeah, meaning it will cause garbage values at some point ;)

It's alright though... please add a comment with a link to the issue and a small text explaining the why of the decision to cast int to double and also add something to the readme